static/js/directives/ui/create-external-notification.js

```diff
@@ -103,6 +103,9 @@ angular.module('quay').directive('createExternalNotification', function () {
         ApiService.createRepoNotification(data, params).then(function(resp) {
           $scope.status = '';
           $scope.notificationCreated({'notification': resp});
+        }, function(resp) {
+          $scope.status = 'error';
+          $scope.errorMessage = ApiService.getErrorMessage(resp, 'Could not create notification');
```

Comment:
So we are only performing server-side validation of the input?

Reply:
For the template? Yes. Doing so on the client would be a bit more work.